setup.cfg

```diff
@@ -0,0 +1,23 @@
+[metadata]
+name = vantiqsdk
+version = 0.9.9
```

Comment:
Once this & the two in extension sources are approved, will update this version to 1.something & publish to PyPI.  This will happen before merge.  Right now, I just publish to test.pypi.org

Reply:
So, I assume we'll need to bump this anytime we do a release.  Might want to document that somewhere.  Be nice if we could figure out a way to generate the release numbers here based on tags in Git like we do for server/UI/docs, but that's not an immediate priority.

Reply:
May want to create something like "releasing.md" which contains a description of the release process.

Reply:
Yes, I think I have a few of those to write -- there's a lot of _lore_ around about how things like this are done.

I'll add an issue to look at generating the setup.cfg file from some template, taking the version from build.gradle or something.  That's more appropriate for us, albeit less Pythonesque for those in that camp.  But, realistically, we'll be the ones doing it, so...

I think I did something like that in the K8sWorker stuff. Will make more sense to change It there.  However, yes, it does need to get updated each time its published -- just like the Java/node/anything-else SDK.